diff --git a/node_modules/winston-logzio/lib/winston-logzio.d.ts b/node_modules/winston-logzio/lib/winston-logzio.d.ts
index d25d5ac..df4e82e 100644
--- a/node_modules/winston-logzio/lib/winston-logzio.d.ts
+++ b/node_modules/winston-logzio/lib/winston-logzio.d.ts
@@ -2,7 +2,7 @@ import { ILoggerOptions } from "logzio-nodejs";
 import Transport from "winston-transport";
 
 declare namespace LogzioWinstonTransport {
-  interface IWinstonLogzioLoggerOptions extends ILoggerOptions {
+  interface IWinstonLogzioLoggerOptions extends ILoggerOptions, Transport.TransportStreamOptions {
     name?: string;
     level?: string;
   }
